components/Repo/Add.js

```diff
@@ -127,14 +177,65 @@ class RepoAdd extends Component {
           }}
         >
           <div {...styles.select}>
-            <Dropdown
-              label='Vorlage'
-              items={templateOptions}
-              value={template}
-              onChange={item => {
-                this.setState({ template: item.value })
-              }}
-            />
+            {data ? (
+              <Loader
+                error={data.error}
+                loading={data.loading}
+                render={() => (
+                  <Autocomplete
+                    label='Vorlage'
+                    value={templateItem}
+                    filter={templateFilter}
+                    items={templateOptions
+                      .concat(
+                        (data?.reposSearch?.nodes || []).map(repo => ({
+                          value: repo.id,
+                          text: repo.latestCommit.document.meta.title
+                        }))
+                      )
+                      .filter(
+                        ({ text }) =>
+                          !templateFilter ||
+                          text
+                            .toLowerCase()
+                            .includes(templateFilter.toLowerCase())
+                      )}
+                    onChange={item => {
+                      this.setState({
+                        templateItem: item,
+                        template: item.value,
+                        templateFilter: ''
+                      })
+                    }}
```

Comment:
```suggestion
                    items={templateOptions
                      .concat(
                        (data?.reposSearch?.nodes || []).map(repo => ({
                          value: repo.latestCommit.document.meta.template,
                          text: repo.latestCommit.document.meta.title,
                          repoId: repo.id
                        }))
                      )
                      .filter(
                        ({ text }) =>
                          !templateFilter ||
                          text
                            .toLowerCase()
                            .includes(templateFilter.toLowerCase())
                      )}
                    onChange={item => {
                      this.setState({
                        templateItem: item,
                        template: item.value,
                        templateRepoId: item.repoId,
                        templateFilter: ''
                      })
                    }}
```

Then simplify `goToEdit`: rm `isRepoId` logic and read `state.templateRepoId`